appservice/src/intTest/groovy/nu/yona/server/AbstractAppServiceIntegrationTest.groovy

```diff
@@ -95,6 +95,7 @@ abstract class AbstractAppServiceIntegrationTest extends Specification
 	{
 		def richard = addRichard(false)
 		def bob = addBob(false)
+		bob.emailAddress = "bob@dunn.com"
```

Comment:
Can't this be inlined in `addBob`? That removes a lot of duplicate code that complicates the tests.